### PR TITLE
refactor(tom): clean up regex to use /b + add help message!

### PR DESCRIPTION
**Pull request by gbeagle**
Cleaned up regex, removed some dead code.
@bernym12

### CODE REVIEW

File: tom/tom.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
 from twisted.words.protocols import irc
 from twisted.internet import reactor, protocol, defer
 from re import search, IGNORECASE
-from random import (
-                    choice,
-                    randint
-                    )
+from random import choice
+
 import random
```
Comment (bernym12) [inline tom/tom.py]: i feel like this is a redundant import. If you only need choice, then delete this line

File: tom/tom.py [hunk 2]
```diff
@@ -29,8 +27,7 @@ class complimentBot(irc.IRCClient):
     owner_name = ""
     currentTime = 0
     default = 'burn berNs'
-    botList = [
-        "dad", "Seahorse", "pointbot", "botProtector",
+    botList = ["dad", "Seahorse", "pointbot", "botProtector",
```
Comment (bernym12) [inline tom/tom.py]: do you plan to use the bot_list.txt file?

File: tom/tom.py [hunk 3]
```diff
@@ -90,40 +82,44 @@ def privmsg(self, user, channel, message):
...
-        elif search(r'(^|\s)+Calix *(\s|$)+', message, IGNORECASE):
+        elif search(r"(\bcalix\b)", message, IGNORECASE):
             self.currentTime = timeRightNow
             self.msg(self.channel, "no sir")
-        elif search(r'(^|\s)+working *(\s|$)+', message, IGNORECASE):
+        elif search(r"(\bworking\b)", message, IGNORECASE):
             self.currentTime = timeRightNow
             self.msg(self.channel, "keep working hard!!!!")
-        elif search(r'(^|\s)+song *(\s|$)+', message, IGNORECASE):
+        elif search(r"(\bfight\ssong\b)", message, IGNORECASE):
```
Comment (bernym12) [inline tom/tom.py]: very nice regex. had no clue.

**Comment by bernym12** [comment]
Also i think you branched off a branch so you've got old commits coming along with this pr

File: tom/tom.py (hunk 2 above)
Comment (gbeagle) [reply tom/tom.py]: That's coming in the future.

File: tom/tom.py (hunk 3 above)
Comment (gbeagle) [reply tom/tom.py]: Thank Jimmy; I looked at memebot since it has to parse all sorts of conditions.

**Comment by gbeagle** [comment]
I opened this PR from master and didn't merge in directly, so Github is complaining.

**Pull request merged**